test/unit/org/apache/cassandra/utils/CassandraGeneratorsTest.java

```diff
@@ -54,7 +54,7 @@ public void partitionerToToken()
     @Test
     public void partitionerKeys()
     {
-        qt().forAll(Gens.random(), toGen(CassandraGenerators.partitioners()))
+        qt().forAll(Gens.random(), toGen(CassandraGenerators.partitioners().map(CassandraGenerators::simplify)))
```

Comment:
flakey test during my first CI run.  `LocalPartitioner` can have very complex type, which leads to tokens too large causing the test to be flakey